archai/nas/operations.py

```diff
@@ -305,6 +315,35 @@ def __init__(self, op_desc:OpDesc, kernel_size:int, padding:int,
     def forward(self, x):
         return self.op(x)
 
+class DepthwiseSeparableConv(Op):
```

Comment:
Please add reference for paper/code from where this comes from. Also, add any notes on differences, other implementations and any perf improvement you have seen. Think of someone looking this code in future and wondering where this comes from.

(same goes to other ops in this file)

Reply:
Great suggestion. Added in doc string for the new ops.